lib/node_modules/@stdlib/blas/base/drot/package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "@stdlib/blas/base/drot",
+  "version": "0.0.0",
+  "description": "Applies a plane rotation.",
+  "license": "Apache-2.0",
+  "author": {
+    "name": "The Stdlib Authors",
+    "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+  },
+  "contributors": [
+    {
+      "name": "The Stdlib Authors",
+      "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+    }
+  ],
+  "main": "./lib",
+  "browser": "./lib/main.js",
+  "gypfile": true,
+  "directories": {
+    "benchmark": "./benchmark",
+    "doc": "./docs",
+    "example": "./examples",
+    "include": "./include",
```

Comment:
Some of these directories are not present given the absence of native implementations. Ditto for `gypfile` above.